src/components/button-link/button-link.module.css

```diff
@@ -4,56 +4,57 @@
 * Root class applied to all button variations. 
 */
 .root {
-  /* variables */
-  --border-width: 1px;
-
-  /* composition */
-  composes: hds-font-weight-medium from global;
-
-  /* attributes */
-  align-items: center;
-  border-radius: 5px;
-  border-style: solid;
-  border-width: var(--border-width);
-  display: flex;
-  font-size: var(--font-size);
-  gap: 6px;
-  isolation: isolate;
-  justify-content: center;
-  line-height: var(--line-height);
-  padding: var(--padding-top-bottom) var(--padding-left-right);
-  position: relative;
-  text-decoration: underline;
-
-  &::before {
-    border-radius: 8px;
-    border: 3px solid transparent;
-    bottom: -4px;
-    box-sizing: border-box;
-    content: '';
-    left: -4px;
-    position: absolute;
-    right: -4px;
-    top: -4px;
-    z-index: -1;
-  }
-
-  &:focus,
-  &:focus-visible {
-    outline: transparent;
-  }
-
-  &:not(:disabled) {
-    cursor: pointer;
-  }
-
-  &:disabled {
-    background-color: var(--token-color-surface-faint);
-    border-color: var(--token-color-border-primary);
-    box-shadow: none;
-    color: var(--token-color-foreground-disabled);
-    cursor: not-allowed;
-  }
+	/* variables */
+	--border-width: 1px;
+
+	/* composition */
+	composes: hds-font-weight-medium from global;
+
+	/* attributes */
+	align-items: center;
+	border-radius: 5px;
+	border-style: solid;
+	border-width: var(--border-width);
+	display: flex;
+	font-size: var(--font-size);
+	font-family: var(--font-body);
```

Comment:
🗒️ Setting `font-family` was the main change I made here. I don't think Prettier formatted CSS files when we converted from SPACE indentation to TABS 😭